ddln_sitl.proto

```diff
@@ -15,4 +16,6 @@ message UpdatePoseResponse {
 service SitlService {
     rpc SetOrigin(LatLongAlt) returns (Empty);
     rpc UpdatePose(UpdatePoseRequest) returns (UpdatePoseResponse);
+    rpc GetWorldInfo(Empty) returns (WorldInfo);
+    rpc WorldIntersection(WorldIntersectionRequest) returns (WorldIntersectionResponse);
```

Comment:
Consider specifying pre-conditions / current limitations and work-arounds that are necessary for this method to work? That is, the world must be pre-warmed with image requests before you can use the intersections.

Reply:
Added relevant comments. Our PX4 and AirRails flight controllers cannot link to internal proto definition, so in order for them to use collision detection this method must be part of the public interface. This is important for VPS work, especially when flying from Fuchs base. It is relatively high in the mountains, and without this change we hit the "ground" as soon as we fly lower than the base, even if it's hundreds of meters AGL.

Reply:
I think this change is more about the landing. Taking off is probably less of a problem, the ground plane internal to Gazebo is only about 100 meter in size but could be made smaller to disable ground interference other then the take off location.

Reply:
Actually, despite having a `<size>` parameter, it is implemented as an infinite plane in Gazebo. We could still circumvent it by replacing a plane with a cube or another limited shape, but this change is a more robust approach.

@ndl : added a "use at your own risk" comment as discussed in chat